ui/components/institutional/compliance-modal/compliance-modal.scss

```diff
@@ -0,0 +1,9 @@
+.compliance-modal {
+  color: var(--color-text-alternative);
+  font-size: 14px;
```

Comment:
You are overriding the Text components underneath with this and ejecting from our Typography styles. Please don't do this, use appropriate props on the Text components.